src/main/java/uk/gov/justice/probation/courtcaseservice/controller/mapper/CourtCaseResponseMapper.java

```diff
@@ -19,11 +19,9 @@
 
     public static CourtCaseResponse mapFrom(CourtCaseEntity courtCaseEntity, GroupedOffenderMatchesEntity groupedOffenderMatches) {
         return CourtCaseResponse.builder()
-                .caseId(courtCaseEntity.getCaseId())
```

Comment:
Have these just been dropped because we're not using them?

Reply:
No consumer is interested in them so, yes.